examples/dao/Makefile

```diff
@@ -25,6 +23,8 @@ drop-sigma-dao-user:
 	@psql -d $(POSTGRES_DBNAME) -c "DROP OWNED BY $(SIGMA_DAO_UNAME);"
 	@psql -d $(POSTGRES_DBNAME) -c "DROP USER $(SIGMA_DAO_UNAME);"
 
-# execute sigma dao setup file
+# execute sigma dao setup file. Indexer should be running before executing this script
 setup-sigma-dao:
 	@psql -U $(POSTGRES_UNAME) -d $(POSTGRES_DBNAME) -a -f $(SIGMA_DAO_SETUP_FILE)
+# grant read only access.
+	@psql -U postgres -d $(POSTGRES_DBNAME) -c "GRANT SELECT ON TABLE $(SIGMA_DAO_OBJECTS) TO $(SIGMA_DAO_UNAME);"
```

Comment:
can we add this statement to the SIGMA_DAO_SETUP_FILE ?

Reply:
We can move but I would suggest to keep it here because in future if variables value like `POSTGRES_DBNAME `, `SIGMA_DAO_OBJECTS ` changes then we have to do in file as well. Is it ok?

Reply:
I think it's better to make all setup in one file. It's also easier to test.

Reply:
Ok. I have moved it to sql file.